src/Sarif.UnitTests/Converters/AndroidStudioConverterTests.cs

```diff
@@ -345,8 +363,14 @@ public void AndroidStudioConverter_ConvertSarifResult_GeneratesLocationWithOnlyP
 
             LocationInfo locationInfo = GetLocationInfoForBuilder(builder);
 
-            locationInfo.Location.Should().Be(expectedLocation);
-            locationInfo.LogicalLocationComponents.SequenceEqual(expectedLogicalLocationComponents).Should().BeTrue();
+            LocationEqualityComparer.Instance.Equals(locationInfo.Location, expectedLocation)
+                .Should().BeTrue();
+
+            locationInfo.LogicalLocationComponents
+                .SequenceEqual(
+                    expectedLogicalLocationComponents,
+                    LogicalLocationComponentEqualityComparer.Instance)
```

Comment:
It does. I can implement the `ValueComparer` property as well:

```C#
public class LogicalLocationComponent {
     ...
     public static IEqualityComparer<LogicalLocationComponent> ValueComparer => LogicalLocationComponentValueComparer.Instance;
     public bool ValueEquals(LogicalLocationComponent other) => ValueComparer.Equals(this, other);
}
```

(`LogicalLocationComponentValueComparer.Instance` can now be internal)